src/vstart.sh

```diff
@@ -17,8 +17,8 @@ quoted_print() {
 }
 
 debug() {
-  if [ -w /dev/tty -a ! -t 2 ]; then
-    "$@" | tee /dev/tty >&2
+  if [ -w /dev/stderr -a ! -t 2 ]; then
+    "$@" | tee /dev/stderr
   else
```

Comment:
I'm not sure what jenkins is doing here to make `/dev/tty` visible but not writeable. In any case, to unblock your testing, just make `debug()` unconditionally call `"$@" > &2`